src/core/packages_manager.rs

```diff
@@ -27,16 +27,23 @@ impl PackagesManager {
     }
 
     pub fn remove_package(&mut self, package_name: &str) -> Result<(), String> {
-        // Find the package URI by package name
         let package_uri = self
             .packages
             .iter()
             .find(|(_uri, pkg)| pkg.name() == package_name)
             .map(|(uri, _pkg)| uri.clone());
 
         if let Some(uri) = package_uri {
-            // Remove the package from the HashMap using the URI
             self.packages.remove(&uri);
+
+            let provider =
+                get_provider(&uri, &self.settings.runtime_dir).map_err(|e| e.to_string())?;
+            let package_dir = provider.get_installation_path();
+
+            if let Err(e) = std::fs::remove_dir_all(package_dir) {
```

Comment:
can we make sure it will only remove in the runtime folder and not in other locations?
i'm asking because it's possible to add packages from any local path on a computer, for example:
`cocmd install /some/path/in/computer`
and if you have `cocmd.yaml` there it will add it. 
i think we need to make sure the provider is not of type [LocalPackageProvider](https://github.com/cocmd/cocmd/blob/master/src/package_provider/local.rs#L32)

Reply:
thanks!